src/Analyzers/CSharp/CodeFixes/UseParameterNullChecking/CSharpUseParameterNullCheckingCodeFixProvider.cs

```diff
@@ -60,6 +60,33 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                         var parameterReferenceSyntax = nullCoalescing.Left;
                         editor.ReplaceNode(nullCoalescing, parameterReferenceSyntax.WithAppendedTrailingTrivia(SyntaxFactory.ElasticMarker));
                         break;
+                    case IfStatementSyntax { Else.Statement: BlockSyntax { Statements: var statementsWithinElse } } ifStatementWithElseBlock:
+                        {
+                            var parent = (BlockSyntax)ifStatementWithElseBlock.Parent!;
+                            var statements = parent.Statements;
+                            var indexOfIfStatement = statements.IndexOf(ifStatementWithElseBlock);
+                            using var _1 = ArrayBuilder<StatementSyntax>.GetInstance(capacity: statements.Count + statementsWithinElse.Count - 1, out var newStatements);
+
+                            for (var i = 0; i < indexOfIfStatement; i++)
+                            {
+                                newStatements.Add(statements[i]);
+                            }
+                            foreach (var statementWithinElse in statementsWithinElse)
+                            {
+                                newStatements.Add(statementWithinElse.WithPrependedLeadingTrivia(SyntaxFactory.ElasticMarker));
+                            }
+                            for (var i = indexOfIfStatement + 1; i < statements.Count; i++)
+                            {
+                                newStatements.Add(statements[i]);
+                            }
+
+                            parent.Statements.ReplaceRange(ifStatementWithElseBlock, newStatements);
```

Comment:
this line appears to have no purpose afaict.

Reply:
That's correct. Removed.